templates/etc.httpd.conf.httpd.conf.j2

```diff
@@ -117,15 +118,16 @@ AccessFileName          .htaccess
 
 # Server Status report - internal only (be careful not to allow NAT ranges)
 ExtendedStatus on
-
-<Location /server-status>
-    SetHandler server-status
-    Order Deny,Allow
-    Deny from all
-{% for address in apache_allow_server_status_addresses %}
-    Allow from {{ address }}
-{% endfor %}
-</Location>
+<VirtualHost *:{{ apache_server_status_listen_port }}>
```

Comment:
This would need to be parameterized since you can't have overlapping VirtualHosts on the same port without a distinct ServerName directive.